xbmc/pvr/guilib/guiinfo/PVRGUIInfo.cpp

```diff
@@ -447,6 +447,10 @@ bool CPVRGUIInfo::GetListItemAndPlayerLabel(const CFileItem* item,
       case LISTITEM_DIRECTOR:
       case LISTITEM_CHANNEL_NUMBER:
       case LISTITEM_PREMIERED:
+      case LISTITEM_PARENTAL_RATING:
+      case LISTITEM_PARENTAL_RATING_CODE:
+      case LISTITEM_MPAA: // For compatibility with other skins.
+      case LISTITEM_PARENTAL_RATING_ICON:
         break; // obtain value from channel/epg
```

Comment:
As you introduced separate new fields in PVR_TIMER in the PVR add-on API for these values, it makes no sense to obtain them from timer's channel/epg tag. Either you keep this implementation and remove the new fields in PVR_TIMER or you change the implementation here to actually use the new fields.